syntax				= "proto3";
option optimize_for = CODE_SIZE;

message Cfg_Scene_MonsterGenerator
{



		uint32 idmap	 = 1;
		uint32 idx		 = 2;
		uint32 active	 = 3;
		uint32 shape	 = 4;
		float  x		 = 5;
		float  y		 = 6;
		float  range	 = 7;
		float  width	 = 8;
		uint32 monsterid = 9;  //怪物id
		uint32 per_gen	 = 10; //每次生成多少个
		uint32 wait_time = 11; //每次生成间隔时间
		uint32 gen_max	 = 12; //生成多少个
		uint32 camp_id	 = 13; //阵营ID
		uint64 phase_id	 = 14; //位面ID

}